src/robotide/contrib/testrunner/FileWriter.py

```diff
@@ -41,6 +42,8 @@ class FileWriter:
 
     @staticmethod
     def write(file_path, lines, windows_mode, mode='w'):
+        if not os.path.exists(os.path.dirname(file_path)):
+            os.makedirs(os.path.dirname(file_path))
         if IS_WINDOWS:
```

Comment:
in non-Windows platform, this method will raise exception, because `lines` could be bytes